src/AutoRest.CSharp/Common/Output/Builders/BuilderHelpers.cs

```diff
@@ -203,28 +203,31 @@ public static TypeDeclarationOptions CreateTypeAttributes(string defaultName, st
             return new TypeDeclarationOptions(defaultName, defaultNamespace, defaultAccessibility, isAbstract, false);
         }
 
-        public static MemberDeclarationOptions CreateMemberDeclaration(string defaultName, CSharpType defaultType, string defaultAccessibility, ISymbol? existingMember, TypeFactory typeFactory)
+        public static CSharpType GetTypeFromExisting(ISymbol existingMember, CSharpType defaultType, TypeFactory typeFactory)
         {
-            if (existingMember != null)
+            var newType = existingMember switch
```

Comment:
What if existingMember == null?  ISymbol is interface, it can be null.

It is better to check inside the function, otherwise, this function may be called in incorrect way.

Reply:
No, here `existingMember` is never null. See the signature, it does not have the question mark.
Please note that we enabled nullable static analysis, the compiler ensures that, here `existingMember` is not null.